recipes/pypartmc/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "PyPartMC" %}
+{% set version = "0.0.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyPartMC-{{ version }}.tar.gz
+  sha256: 8fb2d09b01724259e28f7dbbb6e48224852736a451234ef4133b394c405bce27
+
+build:
+  skip: true  # [win or py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  
+
+requirements:
+  build:
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - cmake
+    - ninja
+  host:
+    - pip
+    - python
+    - setuptools-scm
+  run:
+    - numpy
+    - python
+
+test:
+  imports:
+    - PyPartMC
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/PyPartMC/
+  summary: Python interface to PartMC
+  dev_url: https://github.com/open-atmos/PyPartMC/
+  doc_url: https://open-atmos.github.io/PyPartMC/
+  license: GPL-2.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - a-v
+    - slayoo

```

Comment:
Didn't complete the checklist! @slayoo has not consented to be a maintainer.

Reply:
I do confirm.